int64[] LeftCoeff
int64[] RightCoeff
Header header
